src/bookshelf/src/bksw_sec.erl

```diff
@@ -1,7 +1,7 @@
 %% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
 %% ex: ts=4 sw=4 et
-%% @author Eric B Merritt <ericbmerritt@gmail.com>
-%% Copyright 2012 Opscode, Inc. All Rights Reserved.
+%% @author Eric B Merritt <ericbmerritt@gmail.com>          <-- this was (entirely?) rewritten.  who is author now?
+%% Copyright 2012 Opscode, Inc. All Rights Reserved.            and why license?
```

Comment:
The "All Rights Reserved" bit can probably be removed and the Copyright year and company name updated if you want.  Increasingly we've been opting to not include these individual file headers.  But either way is fine I think.

Reply:
Done.